client/app/components/permissions-editor/index.js

```diff
@@ -53,7 +53,10 @@ const PermissionsEditorComponent = {
       $http.post(this.aclUrl, body).success(() => {
         user.alreadyGrantee = true;
         loadGrantees();
-      });
+      })
+        .catch(() => {
```

Comment:
This is too general and might hide other errors. We should:

* Check for status code 403, in which case show the user the error message.
* In any other case, just say that something went wrong.